packages/e2e-test/src/lib/helpers.ts

```diff
@@ -152,7 +152,7 @@ export async function waitForPageWithText(
 
   // The page may not be fully loaded and hence we need to retry.
   let findTextAttempts = 0;
-  const escapedText = text.replace(/"/g, '\\"');
+  const escapedText = text.replace(/(["\\])/g, '\\$1');
```

Comment:
```suggestion
  const escapedText = text.replace(/"|\\/g, '\\$&');
```

Think of all the bytes!

... but tbh could be good to know 😅

Reply:
meh sure :) $& feels so esoteric

Reply:
Which is why we must spread this knowledge across the world!